File: test59_out/test59/InitTestM/InitTestM.js

/*
 * ======== MODULE test59.InitTestM ========
 */

var $$c = function() {
    const InitTestM = {};
    InitTestM.InitTestM = InitTestM
    InitTestM.$name = 'test59.InitTestM';
    InitTestM.pollen$used = 0;
    InitTestM.pollen__run = new $$Ref('test59_InitTestM_pollen__run__E');
    InitTestM.pollen__shutdown = new $$Ref('test59_InitTestM_pollen__shutdown__E');
    InitTestM.targetInit = new $$Ref('test59_InitTestM_targetInit__I');
    InitTestM.$$hostInit = function() {
        var $$text = '';
        $$printf("InitTestM host init\n");
        $$printf("Init order should be: \n");
        $$printf("1. InitTestM2 2. InitTestC 3. InitTestM \n");
        return $$text;
    }
    InitTestM.pollen__uses__ = function() {
        $$bind($units['test59.InitTestM2'], 'pollen$used', true);
    }
    InitTestM.$$privateInit = function() {
    }
    return InitTestM;
}
var $$u = $$c();
$units['test59.InitTestM'] = $$u;
$units.push($$u);

File: test59_out/test59/InitTestM/InitTestM.c

/*
 * ======== string literals ========
 */

static const char test59_InitTestM__s__fileName[] = "InitTestM.p";
static const char test59_InitTestM__s__0[] = "InitTestM host init\n";
static const char test59_InitTestM__s__1[] = "Init order should be: \n";
static const char test59_InitTestM__s__3[] = "InitTestM";
static const char test59_InitTestM__s__2[] = "1. InitTestM2 2. InitTestC 3. InitTestM \n";

/*
 * ======== function definitions ========
 */

void test59_InitTestM_pollen__run__E() {
    test59_InitTestM2_foo__E();
}

void test59_InitTestM_pollen__shutdown__E( uint8 id ) {
}

void test59_InitTestM_targetInit__I() {
}

